sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -76,12 +76,12 @@ internal CryptographyClient(JsonWebKey keyMaterial, TokenCredential credential,
             Argument.AssertNotNull(keyMaterial, nameof(keyMaterial));
             Argument.AssertNotNull(credential, nameof(credential));
 
-            if (string.IsNullOrEmpty(keyMaterial.KeyId))
+            if (string.IsNullOrEmpty(keyMaterial.Id.ToString()))
             {
-                throw new ArgumentException($"{nameof(keyMaterial.KeyId)} is required", nameof(keyMaterial));
+                throw new ArgumentException($"{nameof(keyMaterial.Id)} is required", nameof(keyMaterial));
             }
 
-            _keyId = new Uri(keyMaterial.KeyId);
+            _keyId = new Uri(keyMaterial.Id.ToString());
```

Comment:
why would you turn the URI into a string to turn it into a URI?

Reply:
I wouldn't. There were a couple places where I even fixed old code that did something like this already. This was probably from a series of refactorings (i.e. one atop another) I didn't notice in review. Will fix.